core/templates/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts

```diff
@@ -20,8 +20,7 @@
 // learner-answer-details-data.service.ts is upgraded to Angular 8.
 import { LearnerAnswerDetailsObjectFactory } from
   'domain/statistics/LearnerAnswerDetailsObjectFactory';
-import { LearnerAnswerInfoObjectFactory } from
```

Comment:
Isn't this used in this file in line 103 and 127?

Reply:
I was thinking that it is defined here https://github.com/oppia/oppia/blob/7a88ec6c1e1f29058c7abcb0f640afaf89303ff9/core/templates/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts#L38

Reply:
I think it's because it is imported in UpgradedServices and 'provided' here, but the tests are passing!